modules/machineset-yaml-aws.adoc

```diff
@@ -98,37 +119,52 @@ endif::infra[]
                   values:
                     - <infrastructure_id>-worker-sg <1>
           subnet:
+ifndef::edge[]
             filters:
               - name: tag:Name
                 values:
                   - <infrastructure_id>-private-<zone> <8>
+endif::edge[]
+ifdef::edge[]
+              id: <value_of_PublicSubnetIds>
```

Comment:
To be clear, this should be Local Zone public id created by `Creating a subnet in AWS Local Zones`, not the public id created by installer or VPC CF templates

Reply:
+1. maybe mentioning something like "Step 4" from the Section `Creating a subnet in AWS Local Zones`.
I just realized that there is no item in the legend for this field, it would be nice to add it there. @dfitzmau